src/plugins/vis_builder/public/application/utils/handle_vis_event.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { get } from 'lodash';
+import { ExpressionRendererEvent } from '../../../../expressions/public';
+import { VIS_EVENT_TO_TRIGGER } from '../../../../visualizations/public';
+import { UiActionsStart } from '../../../../ui_actions/public';
+
+export const handleVisEvent = async (
```

Comment:
Just a question, do we need to add error handling here?

Reply:
How do see error handling being added here?

Reply:
Mmm i am mostly thinking if it's possible that `await uiActions.getTrigger()` fail?